### DISCUSSION TITLE
HeAT with pip and Torch 1.9.0

### DISCUSSION THREAD

**Discussion by erayinanc** [helmholtz-analytics/heat#883] [Q&A] [answered]
I was wondering why the setup.py when "pip install heat" requires "torch>=1.7.0, <1.9", but, in the source code torch>=1.7.0 is enough (similarly for "torchvision"). Both PyPI and GitHub have the same versions (1.1.1). 

When using pip, simply modifying setup.py to compile heat would then be alright, or?

Another issue is that the StackOverflow tag "pyheat" is only new and I don't have enough reputation to create this tag. It would be nice to create this tag.

All the best,
Eray

**Comment by coquelin77** [comment, marked as answer]
The code on the github is a bit ahead of the pip release. The change between the version dependency has been made recently. Thank you for pointing this out. In the future we will update the versioning on the master branch when we do the release to show that this will be the next release.

I believe that changing the `setup.py` would not work here. If you install something from PyPi, it installs a per-compilied wheel. If you want to install the newest version of the code, I would recommend cloning the master branch, move into the folder, then run `pip install -e .` to install the package. 

As for the SO tag, I believe that @mtar had created a tag on one point, however, I do not remember what it was off the top of my head. For now, I think that it is best to continue posting issues here. We keep an eye on these boards

Daniel
